### PR TITLE
Adds vomit as an emote

### PR DESCRIPTION
**Pull request by Yoshax**
Because our players are sensible and trustworthy and need to vomit.

### CODE REVIEW

**Comment by redstryker** [comment]
Finally.

**Comment by Techhead0** [comment]
>Because our players [...] need to vomit.

**Comment by Anewbe** [comment]
:confused: Not sure I really want this, but at the same time I've wished for it on occasion.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -589,12 +589,15 @@
 		if("swag", "stopsway")
 			src.animate_tail_stop()
 
+		if("vomit")
+			vomit()
+			return
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: This `return` is trailing and unnecessary.

**Comment by SpadesNeil** [comment]
Does this actually produce puke on the floor? Cuz if so, I see this being abused lol.

**Comment by Yoshax** [comment]
It calls vomit, yes. If it's abused then staff can deal with it.

**Comment by SpadesNeil** [comment]
It seems like very bad practice to put something in that entices people to abuse it, then have the staff punish people for using it.

I don't think this is a good addition. Any time something is added that entices abuse because it's funny or grants an advantage should not be resolved by telling the staff to punish people for using it.

**Comment by SpadesNeil** [comment]
Or, actually, how about requiring a certain nutrition level for vomit to even work?

If you don't have enough food in your gut, you can only dry-heave. This also produces a penalty for abusing vomit, which prevents flagrant abuse without having to be deliberately **intending** to abuse it.

**Comment by Yoshax** [comment]
That is already a thing: https://github.com/PolarisSS13/Polaris/blob/b78be20b75e6ab854970caaae86a16fee7f3ae2b/code/modules/mob/living/living.dm#L818

**Comment by SpadesNeil** [comment]
Oh, there it is! Odd, I've never seen this happen before.

Still I suggest a cooldown to forced vomiting. There should be no reason to vomit more than once every five minutes.

**Comment by Anewbe** [comment]
Yeah, a cooldown on the voluntary emote would be nice.

**Comment by Yoshax** [comment]
That adds more unnecessary code and complexity. It works, and is fine as is. Staff should do their job and deal with 'abuse' rather than hardcoding it.

**Comment by SpadesNeil** [comment]
Or there should not be a feature that entices abuse, but that's just my opinion.

**Pull request merged**